action.yml

```diff
@@ -43,18 +43,16 @@ runs:
       shell: bash
       run: |
         AWS_TOKEN=$(aws codeartifact get-authorization-token --domain ${{ inputs.codeartifact-domain }} --domain-owner ${{ inputs.codeartifact-domain-owner }} --region ${{ inputs.aws-region }} --query authorizationToken --output text)
-        REPO_URL=$(aws codeartifact get-repository-endpoint --domain ${{ inputs.codeartifact-domain }} --domain-owner ${{ inputs.codeartifact-domain-owner }} --repository ${{ inputs.codeartifact-repository }} --region ${{ inputs.aws-region }} --format pypi --query repositoryEndpoint --output text | sed 's/https:\/\///')
+        REPO_URI="${{ inputs.codeartifact-domain }}-${{ inputs.codeartifact-domain-owner }}.d.codeartifact.${{ inputs.aws-region }}.amazonaws.com/pypi/${{ inputs.codeartifact-repository }}/simple"
+        REPO_URL="https://$REPO_URI"
         echo "token=$AWS_TOKEN" >> $GITHUB_OUTPUT
+        echo "repo-uri=$REPO_URI" >> $GITHUB_OUTPUT
         echo "repo-url=$REPO_URL" >> $GITHUB_OUTPUT
-        
 
     - name: Configure PIP
       shell: bash
       run: |
-        pip config set global.index-url https://aws:${REPO_TOKEN}@${REPO_CLEAN_URL}simple/
-      env:
-        REPO_TOKEN: "${{ steps.codeartifact-metadata.outputs.token }}"
-        REPO_CLEAN_URL: "${{ steps.codeartifact-metadata.outputs.repo-url }}"
+        pip config set site.index-url "https://aws:${{ steps.codeartifact-metadata.outputs.token }}@${{ steps.codeartifact-metadata.outputs.repo-uri }}simple/"
```

Comment:
why manually instead of
`aws codeartifact login --tool pip --repository ${{ inputs.codeartifact-domain }} --domain ${{ inputs.codeartifact-domain }} --domain-owner ${{ inputs.codeartifact-domain-owner }}` ?

Reply:
The URL is not changable. 
AWS keeps in in documentation too:
https://docs.aws.amazon.com/codeartifact/latest/ug/python-configure-pip.html

Reply:
and I need URL without `https` + add username